absl/logging/__init__.py

```diff
@@ -709,18 +710,17 @@ def find_log_dir(log_dir=None):
   # Get a list of possible log dirs (will try to use them in order).
```

Comment:
As there is never more than one candidate, I simplified the code and removed the necessity of the loop.

```suggestion
  # Get a possible log dir.
```